core-bundle/src/Resources/contao/themes/flexible/main.css

```
@@ -355,6 +355,9 @@ body:not(.fullscreen) #container {
 	text-overflow:ellipsis;
 	line-height:22px;
 }
+#main_headline span a {
```

```suggestion
#main_headline span a:hover {
```

I find the underline too obtrusive.

---

I would like to leave that to @leofeyer because there surely was an idea behind how links are formatted in the back end and I personally don't care anyway :)

---

I agree with the suggested change. 👍

---

Looks nicer indeed. PR adjusted.